plugins/importtext_procedure.cc

```diff
@@ -772,6 +772,9 @@ struct ImportTextProcedureWorkInstance
             // is configured.
             auto parseColumnName = [&] (const Utf8String & str) -> ColumnPath
                 {
+                    if (str.empty())
+                        return ColumnPath();
```

Comment:
While testing the import text procedure with the structured dataset, I found that a particularity of the dataset was masking a problem with CSV with trailing commas. This addresses it.